components/base/kinematicbase/differentialDrive.go

```diff
@@ -197,10 +175,10 @@ func (ddk *differentialDriveKinematics) GoToInputs(ctx context.Context, desired
 			StartConfiguration: prevInputs,
 			EndConfiguration:   currentInputs,
 		})
-		if positionChange > epsilon {
+		if positionChange > ddk.options.MovementEpsilon {
```

Comment:
This isn't very descriptive of what it does. I'm thinking `HeartbeatDistanceMM` could be an improvement

Reply:
I feel like that doesn't convey the meaning either. How about `MinimumMovementThreshold`